src/policy/fees.cpp

```diff
@@ -195,6 +202,45 @@ TxConfirmStats::TxConfirmStats(const std::vector<double>& defaultBuckets,
     resizeInMemoryCounters(buckets.size());
 }
 
+TxConfirmStats::TxConfirmStats(const std::vector<double>& defaultBuckets, const std::map<double, unsigned int>& defaultBucketMap, const TxConfirmStats& txstatsSource)
+    : buckets(defaultBuckets), bucketMap(defaultBucketMap)
+{
+    decay = txstatsSource.decay;
+    scale = txstatsSource.scale;
```

Comment:
nit: these can go in the initializer list

Reply:
this comment applies to #13990 now